components/combobox/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@spectrum-css/combobox",
+  "version": "1.0.0-alpha.0",
```

```suggestion
  "version": "1.0.0",
```

We can release this as 1.0.0 unless you feel like the state it's in now is likely to change significantly soon in a way that might require a breaking change.

---

I've removed the alpha from the version number: [73d7557](https://github.com/adobe/spectrum-css/pull/1556/commits/73d75570b78e9d501e251f441d86e696022778d5)